AUTHORS

```diff
@@ -3,3 +3,4 @@ Maintainers of GNU Software" (maintain.texi), the section called
 "Recording Changes".
 
 Mikael Djurfeldt <mikael@djurfeldt.com>
+Espen Hagen <espenhgn at gmail.com>
```

Comment:
Please add a newline. (I can't do that myself, since I can't write to your python3 branch and I don't want to mess up history by creating an extra branch.)

Reply:
Newline added! Never mind my outdated comment on "reverse=True". It should not be used anywhere now. 

Great if this could now be squashed and merged. Also, do you plan to make a new upload on PyPi?